library/Icingadb/Widget/BaseServiceListItem.php

```diff
@@ -28,11 +28,30 @@ abstract class BaseServiceListItem extends StateListItem
     protected function createSubject()
     {
         $service = $this->item->display_name;
-        $host = [
-            new StateBall($this->item->host->state->getStateText(), StateBall::SIZE_MEDIUM),
-            ' ',
-            $this->item->host->display_name
-        ];
+        if (isset($this->item->host->icon_image->icon_image)) {
+            $hostIconImage = HtmlElement::create('div', [
+                'class' => 'icon-image inline',
+            ]);
+
+            $hostIconImage->add(HtmlElement::create('img', [
+                'src' => $this->item->host->icon_image->icon_image,
+                'alt' => $this->item->host->icon_image_alt
+            ]));
+
+            $host = [
+                new StateBall($this->item->host->state->getStateText(), StateBall::SIZE_MEDIUM),
+                ' ',
+                $hostIconImage,
```

Comment:
Why are we now showing the host icon in the service list, but not as part of the visual on the left? I thought showing icons there is no option? If it is now, why do we even need the third column? (i.e. Lets show the icons all inline..)

Please explain this mixed solution.

Reply:
As discussed, I removed the inline icon images for the host icon.

Reply:
✅